docs/config/ca-certificate-bundle-for-imagestream-backups.md

```diff
@@ -0,0 +1,1148 @@
+- Calls `processCACertificatesForVelero()` only when imagestream backups are enabled
+- Mounts CA bundle as file and sets `AWS_CA_BUNDLE` environment variable pointing to it
+
+### Why Different from Regular Velero Backups
+
+ImageStream backups require this special CA handling while regular Velero backups do not. See [ImageStream Backup Scope](#imagestream-backup-scope) and [Two CA Certificate Mechanisms](#two-ca-certificate-mechanisms) in Key Concepts for the detailed explanation of why docker-distribution cannot access Velero's BSL `caCert` configuration.
+
+## Implementation Details
+
+### Certificate Collection and Mounting
+
+The implementation is in `internal/controller/`:
+
+#### 1. Certificate Collection (`bsl.go:908-1124`)
+
+```go
+func (r *DataProtectionApplicationReconciler) processCACertForBSLs() (string, error)
+```
+
+**Collection Strategy**:
+
+- Only collects from **AWS provider BSLs** (imagestream backup uses S3)
+- Scans DPA `spec.backupLocations` for CA certificates
+- Scans additional BSLs in namespace (not in DPA spec)
+- Includes system default CA certificates for fallback
+- Validates PEM format and deduplicates certificates
+
+**Output**: ConfigMap `velero-ca-bundle` with concatenated certificates
+
+#### 2. Velero Deployment Configuration (`velero.go:854-916`)
+
+```go
+func (r *DataProtectionApplicationReconciler) processCACertificatesForVelero(
+    veleroDeployment *appsv1.Deployment,
+    veleroContainer *corev1.Container,
+) error
+```
+
+**Deployment Configuration**:
+
+```go
+// Volume mount
+Volume{
+    Name: "ca-certificate-bundle",
+    VolumeSource: corev1.VolumeSource{
+        ConfigMap: &corev1.ConfigMapVolumeSource{
+            LocalObjectReference: corev1.LocalObjectReference{
+                Name: "velero-ca-bundle",
+            },
+        },
+    },
+}
+
+VolumeMount{
+    Name:      "ca-certificate-bundle",
+    MountPath: "/etc/velero/ca-certs",
+    ReadOnly:  true,
+}
+
+// Environment variable for AWS SDK
+EnvVar{
+    Name:  "AWS_CA_BUNDLE",
+    Value: "/etc/velero/ca-certs/ca-bundle.pem",
+}
+```
+
+### When CA Bundle is Created
+
+The CA bundle ConfigMap and volume mount are created based on the `spec.backupImages` field and presence of CA certificates in AWS BSLs:
+
+**Creation conditions**:
+
+1. `spec.backupImages` is `true` or `nil` (defaults to true)
+2. At least one AWS provider BSL has `caCert` configured
+
+**What gets created**: See [Certificate Collection Scope](#certificate-collection-scope) for details on what certificates are collected.
+
+**Disabling**: When `backupImages: false`, no CA processing occurs. See [backupImages Control Field](#backupimages-control-field) and [Disabling ImageStream Backup CA Handling](#disabling-imagestream-backup-ca-handling) for complete behavior details.
+
+### E2E Test Validation
+
+From `tests/e2e/backup_restore_suite_test.go`:
+
+**When `backupImages=true`** (line 638-649):
```

Comment:
Is this reference outdated? The `tests/e2e/backup_restore_suite_test.go` has 469 lines of code...

Also avoid using lines here when referencing file that is not permanent link.

https://github.com/openshift/oadp-operator/blob/ab5b96fecb7d969097c04e0cbb4bff5ffe0cb152/tests/e2e/backup_restore_suite_test.go#L469

Reply:
will follow up